### PR TITLE
Week4

### PR DESCRIPTION
**Pull request by bealecs**
Check out Week 4 folder for the index.html and index.css files, Thanks

### CODE REVIEW

File: ThirdReadMe.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# **Clif Codes**
+
+### *This is serving as my changes to the MarkDown with VS code*
```
Comment (axecopfire) [inline ThirdReadMe.md:3]: Why did you delete the base `README.md`?

**Comment by axecopfire** [comment]
It looks like this branch is based off week 3. Since week 3 could use some updating. I'll hold off on deep diving the styles.

File: ThirdReadMe.md (hunk 1 above)
Comment (bealecs) [reply ThirdReadMe.md:3]: I honestly am not sure how that happened to be honest, I was starting to get confused with all the different branches and creating a read me the new issues I think I overrode the original file on accident. I will re add

**Comment by sotheencourager** [comment]
Hi Clifton,

Great work leading the way with turnaround time and clean code. 

- CSS is looking good
- Animation is good

Please fix the following:
1. Ensure you follow the specified width for media query on mobile

Suggestions for the future (just something to read more about :) )
- Add "alt" attributes to your images https://www.w3schools.com/tags/tag_img.asp It lets users have an idea what the image is if it does not load. Also helps with accessibility
- <span> can be used as an inline element
